### ISSUE TITLE
[Flaking Test] ci-kubernetes-node-kubelet-serial-containerd failed because of too high pod startup latency 50th percentile

### ISSUE THREAD

**Issue by mengjiao-liu** [kubernetes/kubernetes#109140]
### Which jobs are flaking?

https://testgrid.k8s.io/sig-node-release-blocking#node-kubelet-serial-containerd

### Which tests are flaking?

- ci-kubernetes-node-kubelet-serial-containerd
- E2eNode Suite.[sig-node] Density [Serial] [Slow] create a sequence of pods latency/resource should be within limit when create 10 pods with 50 background 
- kubetest.Node Tests

### Since when has it been flaking?
Tue Mar 29 10:32 PDT

### Testgrid link

kubetest.Node Tests

### Reason for failure (if possible)

E2eNode Suite: [sig-node] Density [Serial] [Slow] create a sequence of pods latency/resource should be within limit when create 10 pods with 50 background pods
```
{ Failure test/e2e_node/density_test.go:248
Unexpected error:
    <*errors.errorString | 0xc001b8cb80>: {
        s: "too high pod startup latency 50th percentile: 24.015405996s",
    }
    too high pod startup latency 50th percentile: 24.015405996s
occurred
test/e2e_node/density_test.go:588}
```

e2e.go: Node Tests
```
{  error during go run /go/src/k8s.io/kubernetes/test/e2e_node/runner/remote/run_remote.go --cleanup --logtostderr --vmodule=*=4 --ssh-env=gce --results-dir=/workspace/_artifacts --project=k8s-infra-e2e-boskos-077 --zone=us-west1-b --ssh-user=prow --ssh-key=/workspace/.ssh/google_compute_engine --ginkgo-flags=--nodes=1 --focus="\[Serial\]" --skip="\[Flaky\]|\[Benchmark\]|\[NodeSpecialFeature:.+\]|\[NodeSpecialFeature\]|\[NodeAlphaFeature:.+\]|\[NodeAlphaFeature\]|\[NodeFeature:Eviction\]|\[Feature:DynamicKubeletConfig\]|\[NodeFeature:DevicePluginProbe\]" --test_args=--feature-gates=LocalStorageCapacityIsolation=true --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}" --test-timeout=4h0m0s --image-config-file=/workspace/test-infra/jobs/e2e_node/containerd/image-config-serial.yaml: exit status 1}
```
]E2eNode Suite: [sig-node] Density [Serial] [Slow] create a sequence of pods latency/resource should be within limit when create 10 pods with 50 background pods 
```
{ Failure test/e2e_node/density_test.go:248
Unexpected error:
    <*errors.errorString | 0xc00234a2c0>: {
        s: "too high pod startup latency 50th percentile: 20.009305457s",
    }
    too high pod startup latency 50th percentile: 20.009305457s
occurred
test/e2e_node/density_test.go:588}
```

### Relevant SIG(s)

/sig node
/sig testing

**Comment by pacoxu** [comment]
dup with #108774 which may be fixed by #108958.

**Comment by mengjiao-liu** [comment]
Closing this issue because of duplication.
/close
